apps/aehttp/src/ws_handler.erl

```diff
@@ -64,3 +63,28 @@ broadcast(SenderName, Action, Payload) ->
 
 send_msg(WsPid, SenderName, Action, Payload) ->
     WsPid ! {send, SenderName, Action, Payload}.
+
+create_message(SenderName, Action, Payload) ->
+    MsgWithoutPayload = #{origin => SenderName,
+                          action => Action},
+    EmptyPayload =
+        case Payload of
+            _ when is_list(Payload) ->
+                length(Payload) == 0;
```

Comment:
Since `length(List)` is O(N), it's much better to simply match on the empty list.